src/modules/deposit/entry-point/http/controller.ts

```diff
@@ -20,6 +20,11 @@ export class DepositController {
     return this.depositService.getDeposits(query.status, limit, offset);
   }
 
+  @Get("deposits-v2")
```

Comment:
What do you think of using `v2/deposits` instead? I think it is generally best practice to version APIs like that

Reply:
Yeah, I hesitated to use `v2/deposits` because this implies that the whole API got a new version, but in fact it is only about upgrading a specific endpoint. I made the change tho 👍 It doesn't matter that much anyway because the plan is to replace the old /deposits endpoint with this one